src/src.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
```

Comment:
I'm fine with having a .sln _too_ but is it now the primary build file?

Reply:
I should delete this. Not needed.